### PR TITLE
Fix shadows

### LINKED ISSUE #124: Enable Shadows

### PR DESCRIPTION
**Pull request by robertlong**
Fixes #124

### CODE REVIEW

File: src/client/editor/Editor.js [hunk 1]
```diff
@@ -339,7 +339,8 @@ export default class Editor {
       true
     );
     await this.addUnicomponentNode("Skybox", "skybox", {}, {}, true);
-    await this.addUnicomponentNode("Ground Plane", "ground-plane", {}, {}, true);
+    const ground = await this.addUnicomponentNode("Ground Plane", "ground-plane", {}, {}, true);
+    await this._addComponent(ground, "shadow", { castShadow: true, receiveShadow: true });
```
[the remark this answers is not in the document]
Comment (robertlong) [reply src/client/editor/Editor.js]: Idk, probably not by default.

**Comment by robertlong** [comment]
If we don't get this in by launch, should we merge the change to the ground material in another PR?

**Comment by brianpeiris** [comment]
Yup, makes sense to fix the ground material in #327

**Comment by robertlong** [comment]
Closing. We'll add shadows support in a future PR.

**Pull request closed without merging**